src/server/routes.ts

```diff
@@ -0,0 +1,75 @@
+import express from "express";
+import { collections, products, carousels } from "./data";
+import axios from "axios";
+
+const router = express.Router();
+
+router.get('/:type', (req, res) => {
+    try {
+        const { type } = req.params;
+        let data;
+
+        switch (type) {
+            case 'collections': data = collections; break;
+            case 'products': data = products; break;
+            case 'carousels': data = carousels; break;
+            default: return res.status(404).json({ error: 'Invalid data type requested' });
```

Comment:
```suggestion
            default: return res.status(400).json({ error: 'Invalid data type requested' });
```
Error code should be `400` Bad request type.